docs/changelog/4.2.rst

```diff
@@ -0,0 +1,1147 @@
+Changelogs for 4.2.x
+====================
+
+.. changelog::
+  :version: 4.2.0
+  :released: *unreleased*
+
+  This release *blah blah blah* …
+
+  Major features for this release:
+
+  * Lua records
+  * IXFRDist
+  * API changes
```

Comment:
What are these changes ?

Reply:
This is just a list of suggestions we might talk about in the 4.2.0 announcement, for now I'm mainly interested in your comments on the actual changelog entries.